src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
No reason for these to be protected when not accessed directly by its child classes. Should be private instead.